.github/workflows/test_all.yml

```diff
@@ -21,6 +21,9 @@ jobs:
       - name: check missing __init__ files
         run: build_tools/fail_on_missing_init_files.sh
         shell: bash
+    # concurrency:
```

Comment:
hm, why is this commented?

Would this solve the problem that cancel does not get run when the jobs - that would get cancelled - block the CI budget?

Reply:
I added it for that purpose only, but it does not seem to work. Unless the job starts immediately, i.e. if it's pending for some time becuase runners are busy, those get skipped.

I really don't understand why, asked here: https://github.com/orgs/community/discussions/73503.

Reply:
🤔